### PR TITLE
MySQL charset and Collation

### PR DESCRIPTION
**Pull request by Crell**

### CODE REVIEW

File: src/configuration/services/mysql.md [hunk 1]
```diff
@@ -164,6 +166,26 @@ db:
 
 The above code will increase the maximum allowed packet size (the size of a query or response) to 64 MB.  However, increasing the size of the maximum packet will also automatically decrease the `max_connections` value.  The number of connections allowed will depend on the packet size and the memory available to the service.  In most cases leaving this value at the default is recommended.
 
+## Character encoding
+
+For services created prior to February 2020, the default character set and collation is `latin1`, which is the default in most MySQL/MariaDB.
+
+For services created after February 2020, the default character set is `utf8mb4` and the default collation is `utf8mb4_unicode_ci`.
+
+For MariaDB 10.1 and later and all Oracle MySQL versions, both values can be adjust at the server level in `services.yaml`:
```
Comment (chadwcarlson) [inline src/configuration/services/mysql.md]:
```suggestion
For MariaDB 10.1 and later and all Oracle MySQL versions, both values can be adjusted at the server level in `services.yaml`:
```

File: src/configuration/services/mysql.md [hunk 2]
```diff
@@ -147,11 +147,13 @@ configuration:
 
 If either schemas or endpoints are defined, then no default will be applied and you must specify the full configuration.
 
-## Adjusting MariaDB configuration
+## Adjusting database configuration
 
-For version 10.2 and later, a select few MariaDB configuration properties from the `my.cnf` file are available for adjustment.
+For MariaDB 10.2 and later and all Oracle MySQL versions, a select few configuration properties from the `my.cnf` file are available for adjustment.
```
Comment (chadwcarlson) [inline src/configuration/services/mysql.md]:
```suggestion
For MariaDB 10.1 and later and all Oracle MySQL versions, a select few configuration properties from the `my.cnf` file are available for adjustment.
```

File: src/configuration/services/mysql.md [hunk 3]
```diff
@@ -164,6 +166,26 @@ db:
 
 The above code will increase the maximum allowed packet size (the size of a query or response) to 64 MB.  However, increasing the size of the maximum packet will also automatically decrease the `max_connections` value.  The number of connections allowed will depend on the packet size and the memory available to the service.  In most cases leaving this value at the default is recommended.
 
+## Character encoding
+
+For services created prior to February 2020, the default character set and collation is `latin1`, which is the default in most MySQL/MariaDB.
```
Comment (matthiaz) [inline src/configuration/services/mysql.md]: Can we also add something about how to change this on existing databases/tables if it is set incorrectly?

`ALTER DATABASE db_name CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;`
`ALTER TABLE table_name CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;`

That'll avoid answering this question in support ;-)

Comment (Crell) [reply src/configuration/services/mysql.md]: Eh, I was hoping to delegate that to the MySQL docs, hence the link further on.  I suppose we can include some stock commands though, if no one has an objection.

**Comment by Crell** [comment]
That should address all of the issues noted above.  Note that the MySQL container isn't updated yet, but will be soon.  Tagging as "upcoming version" until that's deployed.  @ludoge, please give us a green light when that's deployed.  Thanks!

**Pull request merged**